script/8_deploySwapAndBridge.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+echo "Instructing the shell to exit immediately if any command returns a non-zero exit status..."
+set -e
+echo "Done."
+
+echo "Navigating to the root directory of the project..."
+cd ../
+echo "Done."
+
+echo "Setting environment variables..."
+source .env
+echo "Done."
+
+echo "Creating $NETWORK directory inside deployment directory..."
+if [ -z "$NETWORK" ]
+then
+      echo "NETWORK variable inside .env file is not set. Please set NETWORK environment variable."
+      exit 1
+else
+      if [ -d "deployment/swap_and_bridge/$NETWORK" ]
+      then
+            echo "Directory deployment/swap_and_bridge/$NETWORK already exists."
+      else
+            mkdir -p deployment/swap_and_bridge/$NETWORK
+      fi
+fi
+echo "Done."
+
+echo "Deploying and if enabled verifying SwapAndBridge smart contract for DIVA protocol..."
+forge script --rpc-url="$L1_RPC_URL" --broadcast --verify --verifier blockscout --verifier-url $L1_VERIFIER_URL -vvvv script/swap_and_bridge/contracts/SwapAndBridge.s.sol:SwapAndBridgeScript $L1_STANDARD_BRIDGE_ADDR $L1_TOKEN_ADDR_DIVA $L2_TOKEN_ADDR_DIVA --sig 'run(address,address,address)' 
```

Comment:
Why is this needed to be passed to the `forge script`?
`$L1_STANDARD_BRIDGE_ADDR $L1_TOKEN_ADDR_DIVA $L2_TOKEN_ADDR_DIVA`
Isn't this taken from `.env` file inside `SwapAndBridge.s.sol` deployment` script?

Reply:
I made the `SwapAndBridge.s.sol` agnostic to the protocol for which it is used and just accept the relevant parameter in the `run` function.

Reply:
Why are in this case those variables inside `.env.*` files?

Reply:
Inside `SwapAndBridgeScript` it's not checked if those variables are empty addresses or not.
It would be nice to check that.

On the other hand, all our other deployment scripts are also agnostic to which network they are deployed.
All environment variables are read and checked from `.env` file inside deployment scripts like here:
https://github.com/LiskHQ/lisk-contracts/blob/development/script/contracts/L2/L2Airdrop.s.sol#L26-#L27

In this case `forge script` command has also a smaller list of arguments.

What's your opinion about this?

Reply:
I will add the address check. Regarding the env variables, what I meant is that within the same network, you can deploy this contract for different protocols (e.g. Lido and Diva). Using the .env variables in the contract would still require to have a separate script for each protocol for which you want to deploy the contract (which in general we don't know a priori).

Reply:
the addresses are also check in the actual contract, should we still add the extra check in the script contract?

Reply:
To be unified with other deployment scripts inside this repo, I would say it would be nice if also script would check if valid arguments are passed to the code.

Reply:
Ok, I see.
Same deployment script is used for two different protocols (set of smart contracts).

Reply:
I think I should write a Readme in the swap and bridge folder anyway. Maybe I could explain there